config/config.js

```diff
@@ -6,19 +6,25 @@ NOTICE: Adobe permits you to use, modify, and distribute this file in
 accordance with the terms of the Adobe license agreement accompanying
 it.
 */
-const { awsConfig } = require('./aws')
+const fs = require('fs')
+const path = require('path')
 const { adobeConfig } = require('./adobe')
+
+let awsConfig
+if (fs.existsSync(path.join(__dirname, 'aws.js'))) {
```

Comment:
I wanna double check the logic here - users may or may not provide aws.js. May not, if they have no intention to run our samples with AWS S3 support or use our signing API. Is my understanding correct?

Reply:
Yes, that's correct.